templates/zerver/help/message-info.md

```diff
@@ -1,4 +1,4 @@
-# Viewing information about a message
+# Viewing message information
```

Comment:
I'll just drop the change then considering the PR was already merged, but it's fixed now.